platform/app/services/temporal/activities/update_sync_status_activity.rb

```
@@ -0,0 +1,97 @@
+        activity.logger.error("Sync not found for sync run #{sync_run_id} error: #{e.message}")
+        raise
+      end
+
+      def update_sync_status(status, error_message)
+        status_method = status_mapping[status.to_sym]
+        raise ArgumentError, "Invalid sync status: #{status}" unless status_method
+
+        old_status = @sync.status
+        @sync.public_send(status_method)
+        message = status_message(status, error_message)
+        
```

rubocop (1.68.0)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)